support-workers/src/main/scala/com/gu/WithLoggingSugar.scala

```diff
@@ -5,12 +5,12 @@ import com.typesafe.scalalogging.LazyLogging
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 
-object FutureLogging extends LazyLogging {
+object WithLoggingSugar extends LazyLogging {
 
   implicit class LogImplicitFuture[A](op: Future[A]) {
 
     // this is just a handy method to add logging to the end of any for comprehension
-    def withLogging(message: String): Future[A] = {
+    def withEventualLogging(message: String): Future[A] = {
```

Comment:
I wanted to add a withLogging, and it needed to be clear to the compiler and us which was being called, so I distinguished the naming.